util/bigint/src/hash.rs

```diff
@@ -62,6 +62,8 @@ pub fn clean_0x(s: &str) -> &str {
 	}
 }
 
+extern { fn memcmp(s1: *const i8, s2: *const i8, n: usize) -> i32; }
```

Comment:
and actually, since the libc dependency is necessary to do this correctly, we should just use https://doc.rust-lang.org/libc/x86_64-unknown-linux-gnu/libc/fn.memcmp.html

rather than linking it in ad-hoc